src/datasources/cache/redis.cache.service.ts

```diff
@@ -18,6 +19,7 @@ export class RedisCacheService
     @Inject(LoggingService) private readonly loggingService: ILoggingService,
     @Inject(IConfigurationService)
     private readonly configurationService: IConfigurationService,
+    @Inject(CacheKeyPrefix) private readonly keyPrefix: string,
```

Comment:
Nitpick: What do you think about adding some documentation/comments regarding the purpose of this prefix, its intended usage, or the reason why it is needed? I'm not sure this is the place for it, may be somewhere else. I'm considering that this `prefix` may be not self-explanatory when reading the code.

Reply:
I agree. Added in [20dc54f](https://github.com/safe-global/safe-client-gateway/pull/998/commits/20dc54f91051f3a9afceaaffdaa9cdf19f91a11a)